lib/modules.js

```diff
@@ -74,7 +74,7 @@ export const installBinaryModule = async ({
   }
 
   if (target.asset.endsWith('tar.gz')) {
-    await pipeline(res.body, gunzip(), tar.extract(outFile))
+    await pipeline(streamAsyncIterable(res.body), gunzip(), tar.extract(outFile))
```

Comment:
I feel like I'm doing something wrong here. However, MDN suggests this approach. I didn't find anything in node's stdlib for this 🤔